templates/admin/save_round_results.html.twig

```diff
@@ -0,0 +1,87 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+
+    {% block head_stylesheets %}
+        {{ parent() }}
+        <link rel="stylesheet" href="{{ asset('build/tournament.css') }}">
+    {% endblock %}
+
+    <div class="row list_of_participant">
+        <div class="col-md-2"></div>
+        <div class="col-md-10">
+            <table>
+                <tr>
+                    <td>Місце</td>
+                    <td>Гравець чорними</td>
+                    <td>Гравець білими</td>
+                    <td>Результат</td>
+
+                </tr>
+                {#{% for participant in participants %}#}
+                    <tr>
+                        <td>
+                            {#{{ participant.participantOrder }}#}
+                        </td>
+                        <td>
+                            {{ white_participant.player.firstName }}
+                            {{ white_participant.player.lastName }}
+                        </td>
+                        <td>
+                            {{ black_participant.player.firstName }}
+                            {{ black_participant.player.lastName }}
+                        </td>
+                        <td>
+                            <span>12/0</span>
+                            <p data-toggle="modal" data-target="#exampleModal" class="fa fa-edit"></p>
+                        </td>
+                    </tr>
+                {#{% endfor %}#}
+            </table>
+        </div>
+    </div>
+
+
+    <!-- Modal -->
+    <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Results</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <form method="post" action="{{ path('set_tournament_results', { 'tournamentId': tournament_id }) }}">
+                        Суперник:
```

Comment:
Please use translations. Check other places where non-english phrases are present. Use english in code, but implement translations for these phrases